glm/gtc/matrix_access.inl

```diff
@@ -28,7 +28,7 @@ namespace glm
 	{
 		assert(index >= 0 && index < m[0].length());
 
-		typename genType::row_type Result;
```

Comment:
Euh... this warning is a little dumb because the entire row vector will necessary be initialized in the loop. Oh well, I'll submit an alternative fix.